scripts/utils.js

```diff
@@ -0,0 +1,52 @@
+const meta = {
+  title: encodeURIComponent(document.title),
+  description: encodeURIComponent(document.head.querySelector('meta[name="description"]').content),
+  url: `${window.location.origin}${window.location.pathname}`,
+};
+
+const socials = [{
+  type: 'mail',
+  label: 'Diese Website per E-Mail teilen',
+  href: `mailto:?subject=${meta.title}&amp;body=${meta.description}%0D%0A%0D%0A${meta.url}`,
+  icon: 'mail',
+}, {
+  type: 'clipboard',
+  label: 'Link in die Zwischenablage kopieren',
+  href: meta.url,
+  icon: 'content_copy',
+}, {
+  type: 'linkedin',
+  label: 'Diese Website auf LinkedIn teilen',
+  href: `https://www.linkedin.com/shareArticle?mini=true&amp;url=${meta.url}&amp;title=${meta.title}&amp;summary=${meta.description}`,
+  icon: 'linkedin',
+}, {
+  type: 'twitter',
+  label: 'Diese Website auf Twitter teilen',
+  href: `https://twitter.com/intent/tweet?url=${meta.url}&amp;text=${meta.title}`,
+  icon: 'twitter',
+}, {
+  type: 'facebook',
+  label: 'Diese Website auf Facebook teilen',
+  href: meta.url,
```

Comment:
This looks like it is not actually sharing with Facebook, no?

Reply:
good catch ! fixed it here https://github.com/hlxsites/zeiss/pull/25/commits/ff9f8fc11b46a8ac8a3f356d54f6cf0f59371c38